ui/src/app/settings/views/Scripts/ScriptsList.scss

```diff
@@ -0,0 +1,65 @@
+@import "~vanilla-framework/scss/settings_spacing";
+@import "~vanilla-framework/scss/settings_colors";
+
+%heading-icon {
+  background: {
+    image: url("data:image/svg+xml;charset=utf8,%3Csvg xmlns='http://www.w3.org/2000/svg' height='4' width='10' viewBox='0 0 10 4'%3E%3Cpath d='M3.637 3.138c-.518-.365-1.052-.778-1.6-1.238C1.486 1.44.946.948.414.423.273.283.135.14 0 0h1.54c.305.29.62.57.948.846.138.116.277.23.417.34.163.13.328.257.495.38.085.062.17.123.257.184.397.282.935.626 1.315.848h.054c.38-.222.918-.566 1.315-.848.4-.28.79-.583 1.17-.904C7.837.57 8.153.29 8.457 0h1.54c-.134.14-.272.282-.414.422C9.05.948 8.51 1.442 7.963 1.9c-.55.46-1.084.873-1.602 1.238S5.39 3.79 5 4c-.39-.21-.845-.497-1.363-.862z' fill='%23888' fill-rule='evenodd'/%3E%3C/svg%3E");
+    position: center;
+    repeat: no-repeat;
+    size: 100%;
+  }
+  content: "";
+  display: block;
+  flex: 0 0 $sp-medium;
+  height: 0.4rem;
+  margin-top: $sp-small;
+  margin-left: $sp-x-small;
+  vertical-align: middle;
+}
+
+.scripts-list td,
+.scripts-list th {
+  // Script name
+  &:nth-child(1) {
+    flex: 0 0 30%;
+  }
+
+  // Description
+  &:nth-child(2) {
+    flex: 0 0 45%;
+  }
+
+  // Uploaded on
+  &:nth-child(3) {
+    flex: 0 0 15%;
+  }
+
+  // Last edited
+  &:nth-child(4) {
```

Comment:
This looks like it's actually the actions column, setting the width for this one might not be necessary.